learn-in-codemen/04-CSS-Layout/challange_2.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
what is body

Reply:
![Screenshot from 2021-08-06 14-46-00](https://user-images.githubusercontent.com/85335954/132055894-906b77d3-6034-4d8d-a8b1-136ea81fffe9.png)